west.yml

```diff
@@ -28,7 +28,7 @@ manifest:
 
     - name: zephyr
       repo-path: zephyr_alif
-      revision: 0a649c15365376341deb87891f5a470691f8a2f1
+      revision: 56ecb4f540171b8f3e785eee451bb3478c3f83eb
```

Comment:
```suggestion
      revision: pull/10/head
```

Reply:
Please update with the final commit hash once alifsemi/zephyr_alif#10 has been merged.